Code/C++/primenumbers_till_n_nestedforloop.cpp

```diff
@@ -121,3 +121,7 @@ Total Prime Number till N are : 25
 Time Complexity As we are using two nested loop the time complexity of this problem is O(N^2)
```

Comment:
Here also, remove the statement and mention O(n^2) only like this `Time Complexity : O(n^2)`.

Reply:
ook